ACE/ace/ACE.cpp

```diff
@@ -3241,10 +3241,15 @@ ACE::strndup (const wchar_t *str, size_t n)
     continue;
 
   wchar_t *s;
+#if defined (ACE_HAS_ALLOC_HOOKS)
+  ACE_ALLOCATOR_RETURN (s,
+                        (wchar_t *) ACE_Allocator::instance()->malloc (len + 1),
+                        0);
```

Comment:
would prefer a static_cast here instead of a c-style cast

Reply:
Fixed

Reply:
There are other c-style casts in ACE.cpp

Shall I fix them?

Reply:
No need to fix them all at this point, just when you work on code